yarn-project/circuits.js/src/structs/parity/root_rollup_parity_input.ts

```diff
@@ -0,0 +1,39 @@
+import { BufferReader, serializeToBuffer } from '@aztec/foundation/serialize';
+import { type FieldsOf } from '@aztec/foundation/types';
+
+import { RECURSIVE_PROOF_LENGTH } from '../../constants.gen.js';
+import { RecursiveProof } from '../recursive_proof.js';
+import { VerificationKey } from '../verification_key.js';
+import { ParityPublicInputs } from './parity_public_inputs.js';
+
+export class RootParityInput {
```

Comment:
Looks there are two classes the same name `RootParityInput`. There's this one and then there's the one in [yarn-project/circuits.js/src/structs/parity/root_parity_input.ts](https://github.com/AztecProtocol/aztec-packages/pull/6006/files#diff-ba92a849e800c70e66c130be3f19b7dd17669bc3fc74f33a4b5c87c1edbe1a75).
Should this be renamed to match its noir counterpart `RootRollupParityInput`?

Reply:
Good spot, I have deleted this one, it's not used